tests/tasks/tasks/logic/test_loop_task.py

```diff
@@ -204,6 +204,17 @@ def test_subtask_handling(self):
         # check that we handle properly the case of no subtask
         self.task.name += '2'
 
+    def test_renaming(self):
+        """Test renaming the task.
+
+        """
+        subtask1 = CheckTask(name='check', database_entries={'val': 1})
+        self.task.task = subtask1
+        assert subtask1.get_from_database('Test_val') == 1
+
+        self.task.name = 'Test2'
+        assert subtask1.get_from_database('Test2_val') == 1
```

Comment:
This does not test that we actually called the super method. We need to also check that we renamed the entries of the LoopTask itself and not only of the embedded subtask.

Reply:
The renaming of the LoopTask entries is already tested in the renaming of a complextask, do you want to retest it ?

Reply:
(or maybe somewhere else but that's why I did not test it also)

Reply:
I want to be sure we do call the renaming of the ComplexTask, this was the issue after all. But simply checking one value is fine (with a comment explaining why)